content/latest/faq/compatibility.md

```diff
@@ -13,11 +13,18 @@ isTocNested: true
 showAsideToc: true
 ---
 
+## What does API compatibility exactly mean?
+
+API compatibility refers to the fact that the database APIs offered by YugaByte DB servers implement the same wire protocol and modeling/query language as that of an existing database. Since client drivers, command line shells, IDE integrations and other ecosystem integrations of the existing database rely on this wire protocol and modeling/query language, they are expected to work with YugaByte DB without major modifications. 
+
+For example, the [YSQL](../../api/ysql) API is compatible with PostgreSQL. This means PostgreSQL client drivers, psql command line shell, IDE integrations such as TablePlus and DBWeaver and more can be used with YugaByte DB. The same concept applies to [YCQL](../../api/ycql) in the context of the Apache Cassandra Query Language.
```

Comment:
Nit: might be good to put this under a "Note": https://github.com/YugaByte/docs/blob/master/CONTRIBUTING.md#note-box